docs/plugin-maintainers-guide.md

```diff
@@ -0,0 +1,38 @@
+# Plugin Maintainer Guide
+
+## Table of Contents
+
+- [Plugin Maintainer Guide](#plugin-maintainer-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Maintaining and patching an older release line](#maintaining-and-patching-an-older-release-line)
+    - [Patching an older release](#patching-an-older-release)
+
+## Maintaining and patching an older release line
+
+It may be necessary to patch a prior release line of a plugin when users depend on an older, but stable version and while a newer, incompatible, major version of the plugin exists. Typically for these older releases, only major bugs and security issues will need to be remediated. Not every plugin will need this workflow.
+
+This guide will describe the steps needed to release on an older version.
+
+### Patching an older release
+
+When patching an older release, follow the steps below to ensure the correct workflow is applied:
+
+1. Request a `workspace/${workspace}` branch:
+    - Ensure that a branch named `workspace/${workspace}` exists, with appropriate branch protections in place. This branch will be used for patch releases.
+    - The `${workspace}` should correspond to the specific plugin or component you are patching.
+
+2. Reset the `workspace` branch:
+    - Reset the `workspace/${workspace}` branch to the version of the plugin you need to patch.
+    - You can use the autogenerated version tags from previous releases to pinpoint the prior release version to apply the patch.
+
+3. Apply your commits:
+    - Apply the necessary patch fixes or security updates.
+    - Manually bump the version in `package.json` to reflect the new patch version (e.g., from `1.2.3` to `1.2.4`).
+
+4. Run the release workflow:
```

Comment:
Should we highlight what access you need to be able to run the workflow? This could go in a section towards the top. Pretty sure you need to be an Org Member 🤔